feedthefox/devices/migrations/0001_initial.py

```diff
@@ -29,16 +28,28 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('model', models.CharField(max_length=120)),
                 ('manufacturer', models.CharField(max_length=120)),
-                ('imei', models.CharField(default=b'', max_length=17, blank=True)),
                 ('image', models.ImageField(null=True, upload_to=feedthefox.devices.models._get_upload_file_name, blank=True)),
                 ('comment', models.TextField(default=b'', blank=True)),
                 ('builds', models.ManyToManyField(to='devices.Build', blank=True)),
-                ('user', models.ForeignKey(related_name='devices', on_delete=django.db.models.deletion.SET_NULL, blank=True, to=settings.AUTH_USER_MODEL, null=True)),
             ],
             options={
                 'ordering': ('manufacturer', 'model'),
             },
         ),
+        migrations.CreateModel(
```

Comment:
The changes in this migration wont run since migration ``0001`` is already pushed.